spec/src/main/asciidoc/release_notes.asciidoc

```diff
@@ -35,11 +38,12 @@ A full list of changes delivered in the 2.0 release can be found at link:https:/
 
 - OSGi dependencies marked as "provided" (link:https://github.com/eclipse/microprofile-reactive-streams-operators/issues/132[#132])
```

Comment:
Please don't change the previous release notes. All of the updates for the release notes on 2.0 need to be reverted.

Reply:
Not deleted, '-' is for a list